demos-europe/demosplan-core
# bug (refs T29890): orga admin show administer view

**Ticket:** https://yaits.demos-deutschland.de/T29890

Description: As Orga-Admin only you can't access institution tagging over administer ("Verwalten"). This happens if institution tagging is the only permitted menu point in administer. To fix this I added a new entry point route for the orga-admin.

### How to review/test
You could login as User with the role Orga-Admin only and then you should be able to use the administer view.

### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
